cjworkbench/tests/i18n.py

```diff
@@ -9,6 +9,27 @@
 
 
 class TransTest(SimpleTestCase):
+    def test_trans_no_default(self):
```

Comment:
Could you please describe the case where it's okay for a `trans()` call to have no `default`? (I had expected a 500 error and crashed Babel-extract if there's no `default`, so this test expectation comes as a surprise to me.)

Reply:
I wanted to include the ability to use IDs themselves as messages, but indeed this is not what we will do in this project. I will change the behavior and the test.